lib/api/controllers/securityController.js

```diff
@@ -139,16 +159,14 @@ class SecurityController {
   searchRoles(request) {
     checkSearchPageLimit(request, this.kuzzle.config.limits.documentsFetchCount);
 
-    let
+    const
       controllers = request.input.body && request.input.body.controllers,
       from = request.input.args && request.input.args.from && Number(request.input.args.from),
       size = request.input.args && request.input.args.size && Number(request.input.args.size);
 
     return this.kuzzle.repositories.role.searchRole(controllers, from, size)
       .then(response => {
-        response.hits = response.hits.map((role => {
-          return formatProcessing.formatRoleForSerialization(role);
-        }));
+        response.hits = response.hits.map((role => formatProcessing.formatRoleForSerialization(role)));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 103 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)